<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
font: 12pt &quot;Ubuntu&quot;;color:rgb(238, 238, 236)




</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="Basketball">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Ubuntu&quot;; color: black
</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="TeamInformation">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Team Infomation</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="Clasification">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Clasification</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="TournamentBracket">
         <property name="accessibleName">
          <string/>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Tournament Bracket</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="BestPlayers">
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Best Players</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: rgb(252, 233, 79);
selection-background-color: rgb(239, 41, 41);</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoadTeams"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadTeams">
   <property name="text">
    <string>LoadTeams</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
